Cargo.lock

```diff
@@ -19797,7 +19745,7 @@ checksum = "b6e022d0b998abfe5c3782c1f03551a596269450ccd677ea51c56f8b214610e8"
 dependencies = [
  "is-terminal",
  "slog",
- "term 0.7.0",
+ "term",
```

Comment:
Maybe a stupid question, but why did we have a pinned dependency that turned into this one that isn't pinned?

Reply:
It is not pinned. The problem is that different packages in our dependencies depend on different semantic versions of term. So the lock files includes all semantic versions used.